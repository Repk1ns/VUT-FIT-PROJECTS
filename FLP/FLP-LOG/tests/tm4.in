S   A R
A a B R
B x B R
B a C x
B   F  
C x C R
C   E L
C a D R
D x D R
D a C x
E x E L
E a E L
E a B R
 aaaa