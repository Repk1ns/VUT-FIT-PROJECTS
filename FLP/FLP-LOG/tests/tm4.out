S aaaa
 Aaaaa
 aBaaa
 aCxaa
 axCaa
 axaDa
 axaCx
 axaxC
 axaEx
 axEax
 aExax
 Eaxax
 aBxax
 axBax
 axCxx
 axxCx
 axxxC
 axxEx
 axExx
 aExxx
 Eaxxx
 aBxxx
 axBxx
 axxBx
 axxxB
 axxxF
